AGENTS.md

```
@@ -179,6 +179,96 @@ sudo dkms install .
 
 The source in `dkms/src/` has all patches pre-applied and may contain experimental fixes not yet in the patch files.
 
+### DKMS Package Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)